### ISSUE TITLE
Incorrect width after restart desktop app

### ISSUE THREAD

**Issue by iatsyk** [toggl/toggldesktop#1615]
Hi.
Starting from the last update my Toggle desktop application starts like [that](http://prntscr.com/8vfkfv). I mean without bordering to "start menu".
Ok, actually I always put application like [that](http://prntscr.com/8vfmm3). I throw it to the right side of the screen and it resizing to some standard for Windows width, but after restart width become like it was before throwing (wider), upper left corner standstill so it looks like in first screenshot.
Notify me if you need steps to reproduce.

**Comment by amulware** [comment]
Thanks for reporting the problem!

It seems that it does not take the position/size of the taskbar into account properly.

We'll see what we can do!

**Comment by amulware** [comment]
Actually... taking a second look, I think it might be an issue related to dpi scaling (you seem to be using a rather high resolution monitor).

Which version are you using? (you can check in the app's about window)

**Comment by iatsyk** [comment]
App version 7.3.27
Monitor resolution: 1920*1080

BEST Regards.

Ukraine. Vinnytsia
*skype*: windreaver2622
*mobile*:  +380636076872
               +380995225406

**Comment by egoroof** [comment]
I had the same issue at beta channel until I've updated to dev. Don't see this issue now.

**Comment by amulware** [comment]
So far I have not been able to reproduce the problem. I tried a couple of things with different DPI settings, window positions and taskbar configurations.

Could you guys send us a bug report through the app's `send feedback` window? That will include more information, including log files for us to take a look at. **Please include that the info is for _Paul_ (that's me).**

It would be great if you could replicate the bug, and send the feedback right after the app starts with the wrong position. That will make the relevant information much easier to find.

Thanks for your help and patience!

**Comment by iatsyk** [comment]
hmm, maybe I was wrong with describing an issue, here's what I found today:
1) if we moved window somewhere and after that quit app through tray (I mean quit app, but not close window through 'X') - then window current position is stored and after restart it appears in the same position.
2) but if we moved window, then close it through 'X' and then quit app, then position isn't stored and after restart window appears in the previous place (where it was before moving).

Maybe in my situation window somehow was moved out off screen and stored this position, and each next time I've moved it to the right position, and at the end of day, just closed the window and turned off computer which is equals to the second behavior that I described.
